.github/workflows/CI.yml

```
@@ -165,6 +165,17 @@ jobs:
             heffte: 'NoheFFTe'
             hypre: 'NoHYPRE'
             coverage: 'OFF'
+          - distro: 'ubuntu:latest'
```

With https://github.com/ECP-copa/ci-containers/pull/15 merged, we could use
```suggestion
          - distro: 'fedora:nompi'
```
here